pkg/binding/field_copy.go

```diff
@@ -0,0 +1,23 @@
+package binding
+
+import (
+	"bytes"
+	"strings"
+
+	"k8s.io/client-go/util/jsonpath"
+)
+
+func getValuesByJSONPath(obj map[string]interface{}, path_segments ...string) (string, bool, error) {
```

Comment:
this function now returns bool = `false` only when error != nil. Hence, we can drop returning `bool` completely. As mentioned above, the function could return `[]interfaces{}` or something more appropriate, so that we we do not need to deserialize again at the caller.

The library offers `FindResults` function that can be used for that purpose.